cmd/apprepository-controller/controller.go

```diff
@@ -273,14 +273,26 @@ func (c *Controller) syncHandler(key string) error {
 		if errors.IsNotFound(err) {
 			log.Infof("AppRepository '%s' no longer exists so performing cleanup of charts from the DB", key)
 			// Trigger a Job to perfrom the cleanup of the charts in the DB corresponding to deleted AppRepository
-			_, err = c.kubeclientset.BatchV1().Jobs(namespace).Create(context.TODO(), newCleanupJob(apprepo, c.conf), metav1.CreateOptions{})
```

Comment:
`apprepo` is `nil` since the line 269 fails to retrieve it. This was causing a panic in the controller. We just need the namespace and the name at this point.

Reply:
Good catch!  Thank you for pointing this out!